.github/workflows/dest_bigquery.yml

```diff
@@ -0,0 +1,95 @@
+name: Destination Plugin BigQuery Workflow
+
+on:
+  pull_request:
+    paths:
+      - "plugins/destination/bigquery/**"
+      - ".github/workflows/dest_bigquery.yml"
+  push:
+    branches:
+      - main
+    paths:
+      - "plugins/destination/bigquery/**"
+      - ".github/workflows/dest_bigquery.yml"
+
+jobs:
+  plugins-destination-bigquery:
```

Comment:
Not a blocker (we have the same problem with Snowflake) but this won't work for forked PRs, as probably the trust relationship is configured only for the `cloudquery/cloudquery` repo (as it should).
Also forks can't access `secrets`

Reply:
Yeah, I believe this is a limitation right now. We'll need to think of ways to mitigate this at some point, but probably not high urgency.